src/strategies/index.ts

```diff
@@ -27,5 +28,6 @@ export default {
   'yearn-vault': yearnVault,
   moloch,
   pancake,
-  synthetix
+  synthetix,
+  cToken
```

Comment:
The name of the strategy should be lowercase, you can use either `ctoken` or `c-token`